pages/recurring-contributions.js

```diff
@@ -183,7 +192,7 @@ class recurringContributionsPage extends React.Component {
                   mx={2}
                   onClick={() => this.setState({ filter: 'monthly' })}
                 >
-                  <FormattedMessage id="Subscriptions.Monthly" defaultMessage="Monthly recurring contributions" />
+                  <FormattedMessage id="Subscriptions.Monthly" defaultMessage="Monthly" />
```

Comment:
We already have some strings for these ones

```suggestion
                  <FormattedMessage id="Frequency.Monthly" defaultMessage="Monthly" />
```